dspace/src/main/assembly/src-release.xml

```diff
@@ -33,10 +33,35 @@
          <excludes>
             <exclude>**/target/**</exclude>
             <exclude>.*</exclude>
+            <exclude>**/*.zip</exclude>
          </excludes>
          <!-- Ensure line endings in all files are Unix (LF) line endings-->
          <lineEnding>unix</lineEnding>
       </fileSet>
+      <!-- Do not destroy ZIP archives by changing "line" endings! -->
```

Comment:
This could use more explanation as it's a bit of a misleading comment. Perhaps:

Do not destroy source code ZIP archives by changing "line" endings! (e.g. SWORD's example.zip)